src/views/Overview/OverviewQuickLinks.vue

```diff
@@ -1,51 +1,57 @@
 <template>
-  <b-list-group>
-    <b-list-group-item
-      href="#"
-      variant="danger"
-      v-show="logging.entry.Severity === 'Error'"
-      >View 1 high priority event</b-list-group-item
-    >
-    <b-list-group-item>
+  <b-list-group horizontal="md">
+    <b-list-group-item class="flex-fill">
       <dl>
         <dt>BMC time</dt>
         <dd>{{ bmc.Elapsed | date('MMM, DD YYYY HH:MM:SS A ZZ') }}</dd>
       </dl>
     </b-list-group-item>
-    <b-list-group-item>
+    <b-list-group-item class="flex-fill">
       <!-- TODO: add toggle LED on/off funtionality -->
-      <b-form-checkbox v-model="checked" name="check-button" switch>
-        Turn
-        <span v-if="!checked">on</span>
-        <span v-else>off</span> server LED
-      </b-form-checkbox>
+      <dl>
+        <dt>Server LED</dt>
+        <dd>
+          <b-form-checkbox v-model="checked" name="check-button" switch>
+            <span v-if="!checked">on</span>
+            <span v-else>off</span>
+          </b-form-checkbox>
+        </dd>
+      </dl>
     </b-list-group-item>
-    <b-list-group-item
-      href="#"
-      class="d-flex justify-content-between align-items-center"
-    >
+    <b-list-group-item class="flex-fill d-flex align-items-center">
       <!-- TODO: link to SOL -->
-      <span>Serial over LAN console</span>
-      <ChevronRight16 />
+      <b-button
+        href="#"
+        block
+        variant="secondary"
+        class="btn btn-secondary d-flex justify-content-between align-items-center"
```

Comment:
Not sure you need all these classes. `<b-button variant="secondary">` should take care of visual styling. for the icon included in the button, we should double check with jandra, but i think she was ok with keeping button content center aligned